src/gsi/general_read_gfsatm.f90

```diff
@@ -3588,99 +3588,99 @@ subroutine general_read_gfsatm_allhydro_nc(grd,sp_a,filename,uvflag,vordivflag,z
           if ( icount == icm .or. k==nlevs) then
              call general_reload2(grd,g_z,g_ps,g_tv,g_vor,g_div,g_u,g_v,g_q,g_oz, &
                   g_ql,g_qi,g_qr,g_qs,g_qg,icount,iflag,ilev,work,uvflag,vordivflag,g_ni,g_nr)
-      endif
-   enddo ! do k=1,nlevs
-
-   do k=1,nlevs
-      icount=icount+1
-      iflag(icount)=15
-      ilev(icount)=k
-      kr = levs+1-k ! netcdf is top to bottom, need to flip
+          endif
+       enddo ! do k=1,nlevs
```

Comment:
How many spaces are there before the enddo?  
It seems 7. However, I am not so sure.

Reply:
What I tried to do here is to line up the `endif` and `enddo` to their matching `if` and `do` statements.

Reply:
@emilyhcliu Thanks for explanation.